src/pages/_app.tsx

```diff
@@ -1,6 +1,59 @@
-import '@/styles/globals.css'
-import type { AppProps } from 'next/app'
+import type { AppProps } from 'next/app';
+import { ChakraProvider, extendTheme } from '@chakra-ui/react';
+import Header from '@/components/Header';
+import Footer from '@/components/Footer';
+
+/* Dosis fonts */
+    
+import '@fontsource/dosis';
+import '@fontsource/dosis/700.css';
+import '@fontsource/dosis/500.css';
+
+/* Themes */
+
+const theme = extendTheme({
+  colors: {
```

Comment:
@kevinjung13 check the colors system from chakra https://chakra-ui.com/docs/styled-system/theme
we should leverage the colorScheme feature of chakra and provide the full color palette